src/corehost/corehost.cpp

```diff
@@ -154,9 +154,17 @@ int exe_start(const int argc, const pal::char_t* argv[])
     app_path.append(_X(".dll"));
 #endif
 
+    host_mode_t mode;
+#if FEATURE_APPHOST
+    mode = host_mode_t::apphost;
+#elif FEATURE_LIBHOST
```

Comment:
`libhost` should never get here - there's no `libhost` with `main` and there's also no `libhost` which "runs an application".

Reply:
Got rid of passing the mode at all.